feedback.md

```diff
@@ -0,0 +1,19 @@
+# Startrly
+## What We're Looking For
+
+Feature | Feedback
+--- | ---
+**General** |
+Small commits with meaningful commit messages |
+Answered comprehension questions |
+**Code Requirements** |
+Only limited changes to the HTML |
+Page normalized |
+CSS Grid Used for layout |
+Flexbox used for layout |
+Google Font used |
+CSS pseudo selector used |
+Background color |
+DRY CSS Code |
+Attractive page layout and appearance |
+**Overall** |
```

Comment:
I don't really care about "attractive" - for this assignment it's more important that it matches the image.